mmhuman3d/models/heads/pare_head.py

```diff
@@ -0,0 +1,565 @@
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from mmcv.runner.base_module import BaseModule
+from torch.nn.modules.utils import _pair
+
+from mmhuman3d.utils.geometry import rot6d_to_rotmat
+from ..builder import HEADS
+
+BN_MOMENTUM = 0.1
```

Comment:
Should `BN_MOMENTUM` be included in the config?

Reply:
Change it as an argument of the Parehead with a default value of 0.1.